From: Sam Hartman <hartmans@debian.org>
Date: Sun, 10 Aug 2025 09:46:24 -0600
Subject: pam_namespace: adjust paths back to bookworm

The 1.7.0 code uses newer path macros for the secure config_dir.
Adjust back to what bookworm expects.
---
 modules/pam_namespace/pam_namespace.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/modules/pam_namespace/pam_namespace.h b/modules/pam_namespace/pam_namespace.h
index 8a7a66b..b5a0297 100644
--- a/modules/pam_namespace/pam_namespace.h
+++ b/modules/pam_namespace/pam_namespace.h
@@ -86,10 +86,10 @@
 /*
  * Module defines
  */
-#define PAM_NAMESPACE_CONFIG (SCONFIG_DIR "/namespace.conf")
-#define NAMESPACE_INIT_SCRIPT (SCONFIG_DIR "/namespace.init")
-#define NAMESPACE_D_DIR (SCONFIG_DIR "/namespace.d/")
-#define NAMESPACE_D_GLOB (SCONFIG_DIR "/namespace.d/*.conf")
+#define PAM_NAMESPACE_CONFIG (SECURECONF_DIR "namespace.conf")
+#define NAMESPACE_INIT_SCRIPT (SECURECONF_DIR "namespace.init")
+#define NAMESPACE_D_DIR (SECURECONF_DIR "namespace.d/")
+#define NAMESPACE_D_GLOB (SECURECONF_DIR "namespace.d/*.conf")
 #ifdef VENDOR_SCONFIG_DIR
 #define VENDOR_NAMESPACE_INIT_SCRIPT (VENDOR_SCONFIG_DIR "/namespace.init")
 #define VENDOR_PAM_NAMESPACE_CONFIG (VENDOR_SCONFIG_DIR "/namespace.conf")
